resources/styles/components/exercises.less

```diff
@@ -130,8 +170,14 @@
       border: 1px solid @tutor-neutral;
       height: 40px;
       margin-right: 0;
+      margin-left: -1px;
+
```

Comment:
I don't want to remove the border, because I want the one that is active to be the one with the border that shows up over the other borders because the border colors are different.